ndt7-client-cc.cpp

```diff
@@ -85,6 +85,24 @@ void BatchClient::summary() noexcept {
   std::cout << summary.dump() << std::endl;
 }
 
+struct KV {
```

Comment:
```suggestion
struct KeyValueParserState {
```

I think we should be a bit more verbose with naming here.